app/src/main/res/values/strings.xml

```diff
@@ -18,49 +18,49 @@
+    <string name="notification_channel_sync_warnings_desc">Small syncing problems like dismissed changes of read-only collections</string>
     <string name="notification_channel_sync_status">Status messages</string>
     <string name="notification_channel_sync_status_desc">Informational status messages like the sync change summary</string>
 
     <!-- Crash -->
-    <string name="crash_title">EteSync has crashed!</string>
+    <string name="crash_title">EteSync crashed!</string>
     <string name="crash_message">Please send stack trace to developers.</string>
     <string name="crash_email_subject">EteSync Debug Info</string>
-    <string name="crash_email_body">If possible, please include any other relevant information such as what you did to trigger this error.</string>
+    <string name="crash_email_body">If possible, please include any other relevant info, such as what you did before this happened.</string>
 
     <!-- tourguide -->
     <string name="tourguide_title">Did you know?</string>
 
     <!-- startup dialogs -->
     <string name="startup_battery_optimization">Battery Optimization</string>
-    <string name="startup_battery_optimization_message">Android may disable/reduce EteSync synchronization after a few days. To prevent this, turn off battery optimization.</string>
+    <string name="startup_battery_optimization_message">Turn off battery optimization to prevent Android turning off/reducing EteSync syncing after a few days.</string>
     <string name="startup_battery_optimization_disable">Turn off for EteSync</string>
     <string name="startup_dont_show_again">Don\'t show again</string>
     <string name="startup_development_version">EteSync Preview Release</string>
-    <string name="startup_development_version_message">This is a development version of EteSync. Be aware that things may not work as expected. Please give us constructive feedback to improve EteSync.</string>
-    <string name="startup_development_version_give_feedback">Give feedback</string>
+    <string name="startup_development_version_message">This is a development version of EteSync. Be aware that things may not work as expected. Please give us constructive feedback to improve it.</string>
+    <string name="startup_development_version_give_feedback">Provide feedback</string>
     <string name="startup_vendor_specific_bugs">Potential Vendor Bugs</string>
-    <string name="startup_vendor_specific_bugs_message">EteSync has detected you are using an Android version that may contain vendor specific bugs.\nPlease take a look at the FAQ for more information.</string>
+    <string name="startup_vendor_specific_bugs_message">You are using an Android version that may contain vendor specific bugs.\nPlease take a look at the FAQ for more info.</string>
     <string name="startup_vendor_specific_bugs_open_faq">Open FAQ</string>
 
     <!-- AboutActivity -->
-    <string name="about_license_terms">License terms</string>
```

Comment:
I just took a look and this is the title for the actual license content, so I think license terms is better.

Reply:
You are right, I thought it was in the category listing.